### PR TITLE
Prepare Mainnet

### PR DESCRIPTION
**Pull request by h0ngcha0**
Update Guardian addresses and related URLs

### CODE REVIEW

File: configs/alephium/mainnet.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "chainId": 255,
   "groupIndex": 0,
-  "initRewards": 0,
+  "initRewards": 1000,
```
Comment (polarker) [inline configs/alephium/mainnet.json:4]: Do we have a TxScript to topup ALPH rewards?

File: configs/guardian/mainnet.json [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
   "governanceEmitterAddress": "0000000000000000000000000000000000000000000000000000000000000004",
   "initSigners": [
-    "0x2F10c48859bf00F743971aCb287b8876bC9ef7F3",
-    "0x8A3Bc2996485A9e49B104601F40d7b75b79ce3a9",
-    "0xEa7d9D838ea90D2316Ab0c80c8D92719AD5885e8",
-    "0x99068B602684ac2227F21463bf90c29bcBA651Ef"
+    "0x214f15D574A4D7895EEd709F80F7B49E5fE5Ad29",
+    "0x78C7B8dD49CEdd6311f03357bBD7299B4d747852",
+    "0x4B2CbEa0E19473D90e41581d4F39FA44b5fb88fB",
+    "0x9eFb0cd728D59bA13EAbe2f17ceBF9f3AB6989A1"
```
Comment (polarker) [inline configs/guardian/mainnet.json:8]: Are they all different from the testnet addresses?

File: configs/alephium/mainnet.json (hunk 1 above)
Comment (Lbqds) [reply configs/alephium/mainnet.json:4]: Yes, the script is here: https://github.com/alephium/wormhole-fork/blob/master/alephium/contracts/token_bridge_scripts/add_rewards.ral

File: configs/guardian/mainnet.json (hunk 2 above)
Comment (Lbqds) [reply configs/guardian/mainnet.json:8]: Yes, they are all different from the testnet v2 addresses.

File: ethereum/package.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
     "test": "mkdir -p build/contracts && cp node_modules/@openzeppelin/contracts/build/contracts/* build/contracts/ && CHAIN_NAME=ethereum truffle test --network test",
     "migrate": "mkdir -p build/contracts && cp node_modules/@openzeppelin/contracts/build/contracts/* build/contracts/ && truffle migrate --to 6",
     "migrate:goerli": "mkdir -p build/contracts && cp node_modules/@openzeppelin/contracts/build/contracts/* build/contracts/ && truffle migrate --to 6 --network goerli --skip-dry-run",
-    "migrate:mainnet": "mkdir -p build/contracts && cp node_modules/@openzeppelin/contracts/build/contracts/* build/contracts/ && truffle migrate --to 6 --network mainnet --skip-dry-run",
+    "migrate:mainnet": "mkdir -p build/contracts && cp node_modules/@openzeppelin/contracts/build/contracts/* build/contracts/ && truffle migrate --to 6 --network mainnet",
```
Comment (Lbqds) [inline ethereum/package.json:25]: From the [truffle doc](https://trufflesuite.com/docs/truffle/reference/command-line-options/#migrate), we may want to perform the test migration before the real migration, so I removed the `--skip-dry-run` flag. cc @h0ngcha0

**Pull request merged**